setup.py

```diff
@@ -69,15 +79,43 @@ def load_requirements(*requirements_paths):
         )
     return list(requirements)
 
+
+class VerifyVersionCommand(install):
+    """Verify that the git tag matches our version"""
+    description = 'Verify that the git tag matches our version'
+
+    def run(self):
+        tag = os.getenv('CIRCLE_TAG')
+
+        if tag != __version__:
+            info = "Git tag: {0} does not match the version of this app: {1}".format(
+                tag, __version__
+            )
+            sys.exit(info)
+
 # Main ##############################################################
 
+with open("README.rst", "r") as file:
+    long_description = file.read()
 
 setup(
-    name='xblock-flow-control',
+    name='flow-control-xblock',
     version=__version__,
-    description='XBlock - Flow Control',
-    packages=[
-        'flow_control',
+    author='eduNEXT',
+    author_email='technical@edunext.co',
+    description='Flow Control XBlock',
+    long_description=long_description,
+    long_description_content_type='text/x-rst',
+    url='https://github.com/eduNEXT/flow-control-xblock',
+    packages=find_packages(),
+    classifiers=[
+        'Intended Audience :: Developers',
+        'Intended Audience :: Education',
+        'License :: OSI Approved :: GNU Affero General Public License v3',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
```

Comment:
Add the python2 and python 2.7 and Django1.11 classifiers

Reply:
Added.

Reply:
Add also the python 2.7 classifier

Reply:
Added! Missed it during git add